.github/workflows/haskell.yml

```diff
@@ -0,0 +1,172 @@
+name: Haskell CI
+
+defaults:
+  run:
+    shell: bash
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ghc: ["8.6.5"]
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install pkgconfiglite
+      if: matrix.os == 'windows-latest'
```

Comment:
Ach, that's the trick to run a step on a given platform, I was missing it somewhere else :)